### DISCUSSION TITLE
Updating wt32-eth01-bt-proxy

### DISCUSSION THREAD

**Discussion by JustApu** [tube0013/tube_gateways#118] [General]
`To improve Bluetooth reliability and performance, we highly recommend updating wt32-eth01-bt-proxy with ESPHome 2022.12.4 or later. When updating the device to ESPHome 2022.12.4, it is recommended to use a serial cable instead of an over-the-air update to take advantage of the new partition scheme.`

Is there a recommended procedure for updating the Wt32-ETH01 Bluetooth Proxy Kit or is it as simple as plugging it in to a computer with the USB-C cable and use the web-based installer from https://esphome.github.io/bluetooth-proxies/ ?

**Comment by tube0013** [comment]
> simple as plugging it in to a computer with the USB-C cable and use the web-based installer from https://esphome.github.io/bluetooth-proxies/ ?

This is the recommended way
